nemo/collections/asr/data_layer.py

```diff
@@ -39,6 +40,38 @@
 logging = nemo.logging
 
 
+def _process_augmentations(augmenter) -> AudioAugmentor:
+    if isinstance(augmenter, AudioAugmentor):
+        return augmenter
+
+    if not type(augmenter) == dict:
+        raise ValueError("Cannot parse augmenter. Must be a dict or an AudioAugmentor object ")
+
+    augmenter = copy.deepcopy(augmenter)
+
+    augmentations = []
+    for augment_name, augment_kwargs in augmenter.items():
+        prob = augment_kwargs.get('prob', None)
```

Comment:
should we also add a check here that prob is between 0 and 1 ?

Reply:
Good point.

Reply:
Check added.